test/parallel/test-crypto-timing-safe-equal.js

```diff
@@ -0,0 +1,35 @@
+// Flags: --allow_natives_syntax
```

Comment:
Whoops, it's not. Leftover from the test I was extracting it from. I've removed it. Thanks!